CoreFoundation/String.subproj/CFCharacterSet.c

```diff
@@ -1406,7 +1414,9 @@ CFCharacterSetRef CFCharacterSetCreateWithCharactersInString(CFAllocatorRef allo
 
         if (0 == length) {
 	    __CFCSetPutHasHashValue(cset, true); // _hashValue is 0
-	} else if (length > 1) { // Check for surrogate
+	} else if (length > 1) {
+        __CFCharacterSetHash(cset);
```

Comment:
As above, plus the indentation of the `_CFCharacterSetHash` doesn't appear to be indented correctly

Reply:
Will be fixed, thanks for noticing